### DISCUSSION TITLE
General Questions - Entrainment Coefficient

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/fds#12961] [Q&A]
Hi,

Does anyone have a range or an approximate value for the entrainment coefficient of a wall plume or plume really close to a wall (due to the proximity of a burner to a wall)? I think the entrainment coefficient for a top hat profiled plume is roughly 0.15 (plume far away from walls).

Thanks.

**Comment by mcgratta** [comment]
You cannot ask about a coefficient without first telling us what specific empirical correlation the coefficient is for. There are various correlations for plume entrainment.

**Reply by FireStudent321** [reply]
Hi, The empirical correlation takes the form of b=(6/5)(alpha)z. It is based on the ideal plume theory. b stands for radius, alpha stands for entrainment coefficient and z is for plume height above fire source.

**Reply by mcgratta** [reply]
You might want to look at Heskestad's chapter on Fire Plumes in the SFPE Handbook.

**Reply by drjfloyd** [reply]
The conclusion of NUREG-2178 V2 and NIST TN 1934 was that the presence of a wall had no substantial effect on plume temperature or flame height suggesting similar entrainment coefficients.

**Reply by FireStudent321** [reply]
Thanks, drfloyd and mcgratta. 

I've found an article called "Evolution of a wall-attached buoyant plume in confined boxes: Direct numerical simulations, entrainment coefficient and an integral model". The article states that the entrainment coefficient of wall plume was reduced to half of that in the free plume. I was just wondering if I am missing something here.

**Reply by mcgratta** [reply]
I conducted experiments several years ago looking at [wall and corner effects of large fires](https://nvlpubs.nist.gov/nistpubs/TechnicalNotes/NIST.TN.1984.pdf). I did not measure or calculate an entrainment coefficient, but I did measure the temperatures above the fire, both when it was abutting the wall and when it was drawn away from the wall. For the corner fires, there was a noticeable effect on the plume temperatures and flame height. For the wall, not so much, if any. My only comment on the paper that you cite (and I have not read it) is that if direct numerical simulations were conducted, the plume may have been laminar, not turbulent, and maybe the wall had a more profound effect. In my experiment, the flames did not "hug" the wall as one might suspect. Thus, air seemed to penetrate into the base of the plume in at least 3 of the 4 directions.

**Reply by FireStudent321** [reply]
Thanks for the reply. Is it better to assume an entrainment coefficient of 0.15 similar to a free plume?

I've found this link. Link: https://up.codes/s/plume-contact-with-walls. But shouldn't the corner plume diameter be different from wall plume diameter? Also, the equation looks at the diameter. For a wall plume, shouldn't I calculate for half the conical shape of the axisymmetric plume (far away from walls)?

**Reply by mcgratta** [reply]
The simple idea is that for a wall plume, you double the fire area and the heat release rate under the assumption that the fire is "mirrored" in the wall. Then you apply a plume entrainment or temperature correlation to this fictitious fire. This leads to higher temperatures than you would obtain for the original fire. The wall has the effect of limiting the air into the plume, thus retaining the heat higher up. As I said above, we found that this simple theory did not hold particularly well for an actual fire.

**Reply by FireStudent321** [reply]
Thank you for the reply.
